src/components/Dashboard/FilingHistoryList/FutureEffective.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div v-if="filing" class="future-effective-details body-2">
+    <h4>{{_.subtitle}}</h4>
+
+    <p>The {{_.filingLabel}} date and time for {{_.companyLabel}}
+      will be <strong>{{effectiveDateTime}} Pacific Time</strong>.</p>
+
+    <p>If you wish to change the information in this {{_.filingLabel}}, you must
+      contact Registry Staff to file a withdrawal.</p>
+
+    <p>Withdrawing this {{_.filingTitle}} will remove this {{_.filingLabel}}
+      and all associated information, and will incur a $20.00 fee.</p>
+
+    <p v-if="filing.courtOrderNumber">Court Order Number: {{filing.courtOrderNumber}}</p>
+
+    <p v-if="filing.isArrangement">Pursuant to a Plan of Arrangement</p>
+
+    <p>Registries contact information:</p>
+
+    <contact-info class="pt-3" />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { mapState } from 'vuex'
+import { ContactInfo } from '@/components/common'
+import { HistoryItemIF } from '@/interfaces'
+
+@Component({
+  computed: { ...mapState(['entityName']) },
+  components: { ContactInfo }
+})
+export default class FutureEffective extends Vue {
+  readonly entityName!: string
+
+  /** The subject filing. */
+  @Prop({ required: true }) private filing: HistoryItemIF
+
+  /** Data for the subject filing. */
+  private get _ (): any {
```

Comment:
This is interesting. I'm sure there is a good reason, but i'm just curious why name a getter `_` of type `any`.

Reply:
1. I couldn't come up with a good name for this localized data block (`data` works for a while but crashes the tests).
2. I didn't think it was worth an interface to define this data block. It's only used here and it's a pretty small component.

What do you think? Rename? Is there a better data type? Is it OK as is?

Reply:
O ya, fine by me. I was just curious as to why.